src/geo/projection/globe.js

```diff
@@ -41,6 +41,13 @@ export default {
     supportsFreeCamera: true,
     zAxisUnit: "pixels",
     center: [0, 0],
+    unsupportedLayers: new Set([
```

Comment:
We could add the sky layer too if we separate the globe atmosphere rendering from it.